panel/pane/base.py

```diff
@@ -119,7 +116,7 @@ class PaneBase(Reactive):
     # numerical priority is selected. The default is an intermediate value.
     # If set to None, applies method will be called to get a priority
     # value for a specific object type.
-    priority: float | bool | None = 0.5
+    priority: Union[float, bool, None] = 0.5
```

Comment:
Please use this throughout:

```python
from __future__ import annotations
...
    priority: float | bool | None = 0.5
```

Reply:
Thanks I also prefer that, but did not know it was possible.